apps/web/pages/router/embed.tsx

```diff
@@ -1,6 +1,30 @@
+import Head from "next/head";
+
+import type { inferSSRProps } from "@calcom/types/inferSSRProps";
+
+import PageWrapper from "@components/PageWrapper";
+
 import { getServerSideProps as _getServerSideProps } from "../../server/lib/router/getServerSideProps";
 
-export { default } from "./index";
+export default function Router({ form, message }: inferSSRProps<typeof getServerSideProps>) {
```

Comment:
We need to add this component in here due to the fact it removed from "./index" until the embed -> app dir migration is merged

Reply:
let's migrate embed page to app router as well

both index and embed were using same `getServerSideProps`, so we can already migrate this too